src/Raven.Server/Web/Studio/Sharding/Processors/ShardedBucketsHandlerProcessorForGetBucket.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Utils;
+using Raven.Server.Web.Http;
+using Raven.Server.Web.Studio.Processors;
+
+namespace Raven.Server.Web.Studio.Sharding.Processors
+{
+    internal class ShardedBucketsHandlerProcessorForGetBucket : AbstractBucketsHandlerProcessorForGetBucket<ShardedDatabaseRequestHandler, TransactionOperationContext>
+    {
+        public ShardedBucketsHandlerProcessorForGetBucket([NotNull] ShardedDatabaseRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+        protected override async ValueTask<BucketInfo> GetBucketInfo(TransactionOperationContext context, int bucket)
+        {
+            using (var token = RequestHandler.CreateOperationToken())
+            {
+                var shardNumber = ShardHelper.GetShardNumberFor(RequestHandler.DatabaseContext.DatabaseRecord.Sharding, bucket);
+                var cmd = new GetBucketInfoCommand(bucket);
```

Comment:
since this is for debugging, don't we need to point to a specific nodeTag as well?